### PR TITLE
Allow to set/get registry settings from an interface

### PR DESCRIPTION
**Pull request by gforcada**
Fixes:
https://github.com/plone/plone.api/issues/269

### CODE REVIEW

File: src/plone/api/portal.py [hunk 1]
```diff
@@ -281,19 +309,64 @@ def get_registry_record(name=None):
...
+        raise InvalidParameterError(
+            u"The interface parameter has to derive from "
+            u"zope.interface.Interface"
+        )
+
     registry = getUtility(IRegistry)
-    if isinstance(name, str):
+
+    if interface is not None:
+        records = registry.forInterface(interface)
```
Comment (adamcheasley) [inline src/plone/api/portal.py]: This appears to be an exact copy of the code above. Would it not be better to call `get_registry_record` here?

Comment (gforcada) [inline src/plone/api/portal.py]: I was thinking on refactoring it as an external method, but calling ``get_registry_record`` sounds good as well

Comment (gforcada) [inline src/plone/api/portal.py]: Done!

**Comment by mauritsvanrees** [comment]
@adamcheasley When you merge a pull request, can you first check that the package is added in the checkouts.cfg of the relevant plone-coredev branch? Thanks.
I have now done that: https://github.com/plone/buildout.coredev/commit/cf07d530ab5a8f66729a02cd9836080831885a4b
Sorry if I was too fast and you were just about to do that. :-)

**Comment by adamcheasley** [comment]
No problem. I was there ready to commit, but by the time I'd hit the button, you'd already done it!
Thank you.

**Comment by mauritsvanrees** [comment]
Perfect.

**Comment by gforcada** [comment]
Thanks for merging!

**Pull request merged**